nix_update/__init__.py

```diff
@@ -32,6 +32,11 @@ def parse_args() -> Options:
         help="Regex to extract version with, i.e. 'jq-(.*)'",
         default="(.*)",
     )
+    parser.add_argument(
+        "--unstable-version",
```

Comment:
Do you think `--version=unstable` could be used for that? Later `--version` could be re-used again to fetch the latest commit. I can also do the refactoring for that.

Reply:
hmmm, possibly... You mean as an alternative to the current `auto`? Might have to rename `auto` to `stable` then.
Maybe that should be a second PR. Not sure...

Reply:
> hmmm, possibly... You mean as an alternative to the current `auto`? Might have to rename `auto` to `stable` then.
> Maybe that should be a second PR. Not sure...

Right. I have a look.